config/trustedproxy.php

```diff
@@ -23,5 +31,5 @@
      * how many proxies that client's request has
      * subsequently passed through.
      */
-    'proxies' => explode(',', env('TRUST_PROXIES', '')),
+    'proxies' => $proxies,
```

Comment:
Please inline the logic with something like ```'proxies' => strpos(env('TRUST_PROXIES'), ',') ? explode(',', env('TRUST_PROXIES')) : env('TRUST_PROXIES')```

Reply:
as discussed on matrix/irc, the underlying library supports comma-separated strings now. We can pass a string in directly.